sagemaker-autopilot/housing_prices/autopilot_california_housing.ipynb

```diff
@@ -0,0 +1,904 @@
+{
```

Comment:
<p>You could optionally add a JPG image to this notebook showing the feature importance results from one test run.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3026/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>

Reply:
Will do this in a later update once I re-run this notebook